### ISSUE TITLE
Cannot find GPMDP 4.5.0 in Windows Store

### ISSUE THREAD

**Issue by gtrubach** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2989]
Hi!

**OS:** Windows 10 1709. Build 16299.214

**Issue Descriptions:** I see that checkbox of uploading to WinStore is checked [here](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/2950) but I see only 4.4.0 version in store. I tried to manually press 'Get Updates' but 4.5.0 wasn't shown for me.

**Comment by jostrander** [comment]
Confirmed 4.4.0 is still installing via Microsoft Store.

**Comment by CorruptComputer** [comment]
Same here as well, I think it might have to do with Microsoft having to review and approve everything on the store.

**Comment by gtrubach** [comment]
@MarshallOfSound @jostrander Any updates on it? Will we have 4.5.0 in Microsoft Store or we need to wait the next version that will be released in Microsoft Store?

Thanks!

**Comment by boswelja** [comment]
It looks like it's been taken off the Microsoft Store :(

**Comment by MarshallOfSound** [comment]
I removed it manually till I resolve an issue

**Comment by norweeg** [comment]
isn't removing it a little extreme?  I figured you left took it off the store permanently and removed the app version

**Comment by MarshallOfSound** [comment]
> isn't removing it a little extreme?

No, this is not the issue that caused me to remove it.  There is an ongoing thing I am trying to resolve

**Comment by z0oinks** [comment]
I'd also like to know when the UWP version is coming back. The only reason I use Groove is because it can play music even when the screen turns off.

Also, would it be available on Xbox?

**Comment by GryphonSK** [comment]
I also would like to know when it will be coming back to the Windows Store.  I love this app and the Windows Store version works so much better than the older one.

**Comment by mike-mo** [comment]
@MarshallOfSound does this "issue" / "ongoing thing" that caused you to pull this from the Microsoft Store have an issue # we can track, or help with? Or is it a C&D type of problem?

**Comment by fahdriyami** [comment]
I was just setting up a new PC today and was worried that it wasn't in the Store. Good to know that it's not gone forever. Hopefully :p

**Comment by theg00s3** [comment]
Yeah, I concur. Is there a timeline or at least a issue number we can follow?

**Comment by brandonforster** [comment]
@theg00s3 #2989 seems pretty relevant.

**Comment by MichaelTong** [comment]
Still no words on 4.6.0?

**Comment by hiya-tiger** [comment]
I hope to see it back on the Microsoft Store..

**Comment by someniatko** [comment]
@MarshallOfSound any update?

**Comment by Kenya-West** [comment]
Any update? I usually check if the app is available through Microsoft Store.

**Comment by bogdan-calapod** [comment]
For anyone wondering, #3761 has a `Upload to WinStore` checkbox, so I guess it will be back pretty soon. 😄
